ocs_ci/deployment/networkpolicy.py

```
@@ -0,0 +1,66 @@
+
+logger = logging.getLogger(__name__)
+
+
+def _ensure_namespace_exists(namespace):
+    """
+    Ensure the target namespace exists, creating it if necessary.
+    """
+    try:
+        exec_cmd(f"oc get namespace {namespace}")
+    except CommandFailed:
+        logger.info(f"Namespace '{namespace}' does not exist, creating it")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)